operator/src/main/java/org/bf2/operator/operands/KafkaInstanceConfiguration.java

```diff
@@ -390,6 +392,14 @@ public LoggingSuppressionWindow getLoggingSuppressionWindow() {
         public void setLoggingSuppressionWindow(LoggingSuppressionWindow loggingSuppressionWindow) {
             this.loggingSuppressionWindow = loggingSuppressionWindow;
         }
+
+        public String getPrivatePrefix() {
```

Comment:
```suggestion
        public String getPrivateTopicPrefix() {
```
or
```suggestion
        public String getInternalTopicPrefix() {
```
I was initially confused at to what `privatePrefix` meant then I spotted the property.  Not sure if `private` is the right term so maybe `internal` is better, I think that's what we have referred to it as in other contexts.

Reply:
I'm not opposed to a name change, but note that this prefix applies to other "internal" objects besides topics like transaction_id, consumer groups, and client IDs.

Reply:
Ohh, I'd miss read the flow (and scope) of the change I thought this was being driven from `strimzi.authorization.custom-authorizer.partition-counter.private-prefix` but no its providing a value to it.

I was only intending to suggest the `partition-counter` called it a `*TopicPrefix`. 

The more I think about it less I like `internal` as a change from `private`. I thought it was being consistent with other usages such as the canary but I can't find any evidence of that on GitHub. So quite happy to stick with `private`.